schemas/documents/pageLinks.ts

```diff
@@ -0,0 +1,44 @@
+import { DocumentDefinition } from "sanity";
+
+const schema: DocumentDefinition = {
+    name: "pageLinks",
+    title: "Page Links",
+    type: "document",
+    fields: [
+        {
+            name: "header",
+            title: "Header",
+            type: "string",
+            validation: (Rule) => Rule.required(),
+        },
+        {
+            name: "description",
+            title: "Description",
+            type: "text",
+        },
+        {
+            name: "image",
+            title: "Image",
+            type: "image",
+            validation: (Rule) => Rule.required(),
+        },
+        {
+            name: "link",
+            title: "Link",
+            type: "reference",
+            to: [
+                {type: "culturePage"},
```

Comment:
Oo I originally meant a list of strings since that's how we'll have to use them on the site. but this could work too! We could keep a mapping of Sanity schema types to URLs so that editors of the CMS don't have to know for sure what URL to use on the site